src/psyclone/domain/lfric/lfric_builtins.py

```diff
@@ -494,13 +494,30 @@ def get_scalar_argument_references(self):
         literal expression) for any scalar arguments to this Built-In kernel.
 
         :returns: a Reference or PSyIR expression for each scalar kernel \
-            argument.
+                  argument.
         :rtype: list of subclasses of `:py:class:`psyclone.psyir.nodes.Node`
 
         '''
         return [arg.psyir_expression() for arg in self._arguments.args
                 if arg.is_scalar]
 
+    def _create_assignment(self, lhs, rhs):
+        '''
+        Creates an assignment from left-hand- and right-hand-side ArrayReference
+        nodes, then replaces the Builtin with the newly created Assignment
+        node.
+
```

Comment:
You need to document the `lhs` and `rhs` arguments.

Reply:
I've taken the liberty of correcting what you had for rhs (it might not be an array reference).